ReactiveCocoa/RACObjCRuntimeUtilities.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+typedef void (^rac_receiver_t)(void);
```

Comment:
I think this block makes things more confusing—at least on the Swift side. If we don't use a `typedef`, does it come through as `() -> Void`?